src/components/FormInput.jsx

```diff
@@ -1,13 +1,13 @@
 import styled from '@emotion/styled';
 
 export const FormInput = styled.input`
-  width: 268px;
+  width: 248px;
   max-width: 100%;
   height: 53px;
   border-radius: 7px;
   box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
   background: var(--light-gray-1);
-  padding-left: 20px;
+  padding: 0 20px;
   margin: 10px;
   border: none;
   border: 1px solid var(--light-gray-0);
```

Comment:
Border 1px is overwriting border none, can you check what really is needed and remove the other?

Reply:
![image](https://user-images.githubusercontent.com/71513679/161871876-ad49289c-f6ee-4dff-b7b7-b085a299f383.png)

Reply:
THanks